docs/admin/deployment/onion_name.rst

```diff
@@ -0,0 +1,108 @@
+Getting An Onion Name for Your SecureDrop
+-----------------------------------------
+
+What Are Onion Names?
+^^^^^^^^^^^^^^^^^^^^^
+
+Onion names are short, memorable addresses that visitors can use to access an 
+onion service (e.g., a news organization's SecureDrop) using Tor Browser. 
+
+Imagine a SecureDrop instance for a new organization called
+*The New York World* with a .onion address like this:
+
+`sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion`
+
+An onion name for this SecureDrop instance could be:
+
+`nyworld.securedrop.tor.onion`
+
+The general format for a SecureDrop onion name is:
+
+`<organization>.securedrop.tor.onion``
```

Comment:
Stray backtic. Note that we'll want to use double backtics for preformatted text. I have a slight preference for preformatting over italics here, but either is fine with me.

Reply:
Thank you! I also prefer the preformatting, I just keep forgetting that RST uses two backticks for it. Fixed :)